### PR TITLE
Add script to fetch the latest binary versions

### PR DESCRIPTION
**Pull request by Roming22**
Signed-off-by: Romain Arnaud <rarnaud@redhat.com>

### CODE REVIEW

File: developer/images/dependencies-update/content/bin/tasks/update_binaries.sh [hunk 1]
```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Pipeline Service Authors.
```
Comment (ramessesii2) [inline developer/images/dependencies-update/content/bin/tasks/update_binaries.sh:3]: Shall we put  2023 for new scripts?

**Review by ramessesii2 — APPROVED** [review]
Looks good to me. One minor comment though.

File: developer/images/dependencies-update/content/bin/tasks/update_binaries.sh (hunk 1 above)
Comment (Roming22) [reply developer/images/dependencies-update/content/bin/tasks/update_binaries.sh]: Good catch! Fixed.

**Review by bnallapeta — APPROVED** [review]
As discussed, there will be a PR to follow this one to actually make changes in the tekton yaml to call this script.

**Pull request merged**